examples/design/easter/script.js

```diff
@@ -102,11 +97,6 @@ const rabbit = new WHS.Model({
     })
   ],
 
-  material: new THREE.MeshLambertMaterial({
```

Comment:
@hirako2000 Unused one. It takes material from model because `useCustomMaterial` is `false` by default.